kpaamcamcommands/analyze.py

```diff
@@ -0,0 +1,135 @@
+"""
```

Comment:
This is the analysis code wrapped in a cldfbench subcommand, so it can be 
- discovered via `cldfbench -h`, 
- inspected via `cldfbench lowerfungom.analyze -h`
- and run via `cldfbench.lowerfungom`